clap-v3-utils/src/input_parsers/mod.rs

```diff
@@ -21,12 +21,18 @@ pub mod signer;
     since = "1.17.0",
     note = "Please use the functions in `solana_clap_v3_utils::input_parsers::signer` directly instead"
 )]
+#[allow(deprecated)]
 pub use signer::{
     pubkey_of_signer, pubkeys_of_multiple_signers, pubkeys_sigs_of, resolve_signer, signer_of,
     STDOUT_OUTFILE_TOKEN,
 };
 
 // Return parsed values from matches at `name`
+#[deprecated(
+    since = "1.19.0",
```

Comment:
These need all updating to v2.0.0, unless we are planning to backport to v1.18.

Reply:
Thanks, fixed!